src/style.css

```diff
@@ -2,0 +1,39 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. If you have specified a custom font for your `.stars__star` class in your HTML, make sure to add a fallback font here.